### DISCUSSION TITLE
Reorganising our discussions

### DISCUSSION THREAD

**Discussion by Daeraxa** [pulsar-edit/.github#168] [Discussions Meta]
So people may (or may not be) aware of a feature that has *finally* hit the public GitHub Discussions which is the ability to have subcategories (or sections) - you can find the GitHub blog post here - https://github.blog/changelog/2023-04-17-organize-discussions-categories-into-sections/.

Now the question is do we want to use this here? I'd love for this space to become more popular and I think a big part of it is having a forum structure that makes sense.

Currently our categories are as such:
- Announcements
- Community Packages
- Core Application & Packages
- Discussions Meta
- Documentation & Website
- General
- Ideas
- Off Topic
- Support
- User Interface

Personally I don't feel this is great and that we can do things to improve this. I'd like to set an overall "theme" for each section that could help with this.

For example:

- Development (For help and assistance developing/hacking etc. on Pulsar)
  - Community Packages
  - Core Application
  - Core Packages
- Support (For community help and assistance - not bug/issue reporting) 
  - Pulsar
  - Packages
  - Other
- Feature requests (Feature request, enhancement discussion, ideas for new packages etc.)
  - Pulsar
  - Packages
- Lounge (For anything and everything else)
  - General
  - Off-topic
  - Meta

That is just a quick idea off the top of my head but I'd be interested to hear if people have any better ideas of what we could do here.

**Comment by confused-Techie** [comment]
So I'd totally be a fan of making our discussions even better, but this idea does have a few questions.

* Where would we want to put the pre-existing discussions? Would we try to move them to the appropriate categories? 
* This has no section for bug reports. Which I love, since that's not the point of discussions, but we have seen it used that way. Do you think we should address this at all? Like literally put in the 'Support' section that this isn't for bug reports

**Reply by Daeraxa** [reply]
>     * Where would we want to put the pre-existing discussions? Would we try to move them to the appropriate categories?

Yup, that would be the plan.
 
>     * This has no section for bug reports. Which I love, since that's not the point of discussions, but we have seen it used that way. Do you think we should address this at all? Like literally put in the 'Support' section that this isn't for bug reports

Two ways of dealing with it, we can set descriptions for the categories pretty easily but the other method is that we provide templates/forms for the areas which need stricter moderation. See https://docs.github.com/en/discussions/managing-discussions-for-your-community/creating-discussion-category-forms - where we can try to make it more obvious.

**Reply by confused-Techie** [reply]
I see, yeah using both of these methods sounds awesome to reduce noise in discussions. And otherwise good to know we are on the same page about how to handle existing discussions. So in that case I'm totally on board with this idea, and willing to help out once we decide how we want them organized

**Reply by confused-Techie** [reply]
Which FYI the existing example organization you put above seems perfect to me. Except maybe in the `Development` category we add a subcategory of `Finding Resources` since quite often it seems people just need help finding something, especially if they don't know if this is within the main editor, or a community package or something that transcends those. For example, where can I find the classes to target to create a theme?

**Reply by Daeraxa** [reply]
I like the idea but I think the name is somewhat ambiguous. We can always make a generic "hacking" section? Or a themes section specifically.

**Reply by confused-Techie** [reply]
Yeah that's fair. Think you are right with a `Hacking` section, as that should be obvious enough
